backend/src/main/java/com/htecgroup/SkyNest/service/EmailService.java

```diff
@@ -0,0 +1,5 @@
+package com.htecgroup.skynest.service;
+
+public interface EmailService {
```

Comment:
Btw I just notice.. SkyNest is deprecated package so needs to be removed , so comments are all applicable for new files